packages/algob/src/lib/tx.ts

```diff
@@ -249,7 +249,7 @@ export async function executeTransaction (
     }
     const confirmedTx = await deployer.sendAndWait(signedTxn);
     console.log(confirmedTx);
-    await registerCheckpoints(deployer, txns, txIdxMap);
+    if (deployer.isDeployMode) { await registerCheckpoints(deployer, txns, txIdxMap); }
```

Comment:
Shall we add a test?

Reply:
yes, thanks
tests are missing for deploy and update as well,
let's add them